infra/terraform/test_fixtures/terraform-google-iam.tf

```diff
@@ -0,0 +1,80 @@
+locals {
+  // Roles required by CI service accounts to run the IAM
+  // integration tests.
+  iam_required_roles = [
```

Comment:
The following roles are missing:

Compute Admin
Compute Network Admin
Folder Admin
Folder IAM Admin
Service Account Admin
Owner
Project IAM Admin
Service Account Admin

Reply:
Addressed in 2c74eae Add missing roles for IAM fixture